tus_storagehandler/api/elixircloud/__init__.py

```diff
@@ -0,0 +1 @@
+"""Elixircloud API package."""
```

Comment:
"ELIXIR Cloud" instead of "Elixircloud". More generally, please make sure ELIXIR is always fully capitalized (unless it's in a package name or variable etc, where only lowercase chars should be used).